ravenframework/Models/PostProcessors/SparseSensing.py

```diff
@@ -68,14 +81,92 @@ class cls.
+    goal.addSub(width)
+    height = InputData.parameterInputFactory("height", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""Height of the ellipse""")
+    goal.addSub(height)
+    angle = InputData.parameterInputFactory("angle", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""Angle of rotation of the ellipse""")
+    goal.addSub(angle)
+    x1 = InputData.parameterInputFactory("x1", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""X co-ordinate of one of the points that defines the line""")
+    goal.addSub(x1)
+    x2 = InputData.parameterInputFactory("x2", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""X co-ordinate of the other point that defines the line""")
+    goal.addSub(x2)
+    y1 = InputData.parameterInputFactory("y1", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""Y co-ordinate of one of the points that defines the line""")
+    goal.addSub(y1)
+    y2 = InputData.parameterInputFactory("y2", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""Y co-ordinate of the other point that defines the line""")
+    goal.addSub(y2)
+    h = InputData.parameterInputFactory("h", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""X coordinate of the vertex of the parabola we want to be constrained""")
+    goal.addSub(h)
+    k = InputData.parameterInputFactory("k", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r""" Y coordinate of the vertex of the parabola we want to be constrained""")
+    goal.addSub(k)
+    a = InputData.parameterInputFactory("a", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""X coordinate of the focus of the parabola""")
+    goal.addSub(a)
+    xy_coords = InputData.parameterInputFactory("xy_coords", contentType=InputTypes.IntegerListType,   ##What is the type for an array?
```

Comment:
Hm, some of the comments like "What is the type for an array?" look like they may be out of date.